src/ui/handler/scroll_zoom.js

```diff
@@ -155,6 +163,16 @@ class ScrollZoomHandler {
     wheel(e: WheelEvent) {
         if (!this.isEnabled()) return;
 
+        if (this._requireCtrl) {
+            if (!e.ctrlKey && !this.isZooming()) {
```

Comment:
The behavior looks great. For me though, it doesn't work because it's overridden by the ctrl + wheel accessibility feature. I'm not sure how common this is.

<img width="668" alt="Screen Shot 2021-09-20 at 11 24 43 AM" src="https://user-images.githubusercontent.com/572717/134054463-2a5343ac-0f7e-4b99-b63a-1c4394dbb932.png">

This might favor the alternative I've seen elsewhere of allowing ctrl + wheel *or* meta + wheel so that a user's accessibility settings don't block this interaction.